src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help-script.html

```diff
@@ -9,4 +9,42 @@
     Otherwise the system default shell will be run, using the <code>-xe</code> flags
     (you can specify <code>set +e</code> and/or <code>set +x</code> to disable those).
     </p>
-</div>
\ No newline at end of file
+    <p>A few examples of usage include:</p>
+
+    <b>Creating an output folder</b>
+    <p><code>sh "mkdir -p output"</code></p>
```

Comment:
Use single quotes instead of double quotes.  We don't want to risk [groovy interpolation](https://groovy-lang.org/syntax.html#_string_interpolation) except when we specifically need it.
```suggestion
    <p><code>sh 'mkdir -p output'</code></p>
```